app/services/nebraska_hours_calculator.rb

```diff
@@ -20,13 +20,18 @@ def calculate_hours
   end
 
   def round_hourly_to_quarters(duration)
-    adjusted_duration = if duration <= (5.hours + 45.minutes)
-                          duration
-                        elsif duration > 10.hours && duration <= (14.hours + 45.minutes)
-                          duration - 10.hours
-                        else
-                          0.minutes
-                        end
-    (adjusted_duration.in_minutes / 15.0).ceil * 15 / 60.0
+    (adjusted_duration(duration).in_minutes / 15.0).ceil * 15 / 60.0
+  end
+
+  def adjusted_duration(duration)
+    if duration <= (5.hours + 45.minutes)
+      duration
+    elsif duration > 10.hours && duration <= (14.hours + 45.minutes)
+      duration - 10.hours
+    elsif duration > 24.hours
```

Comment:
@JonErvin this updates the logic based on the new information about more than 24-hours care